src/providers/erogames-api/api.js

```diff
@@ -0,0 +1,117 @@
+import errorCodes from './error-codes'
+
+const registrationEndpoint = 'https://erogames.com/api/v1/register'
+
+const buildFetchOptions = body => ({
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify(body)
+})
+
+const parseResponseForError = (possibleErrorsByPriority, response) => {
+  let responseError = errorCodes.NO_ERROR
+  possibleErrorsByPriority.some(possibleError => {
+    if (response.errors && possibleError in response.errors) {
+      responseError = possibleError
+      return true
+    }
+    return false
+  })
+  return responseError
+}
```

Comment:
Un peu de code golf (pas testé, pas formatté):
```suggestion
const parseResponseForError = (possibleErrorsByPriority, response) => {
  return possibleErrorsByPriority.find(e => response.errors && e in response.errors) || errorCodes.NO_ERROR
}
```